packages/design-system/src/Button/Button.styles.tsx

```diff
@@ -0,0 +1,246 @@
+import styled, { css } from "styled-components";
+import {
+  ButtonContentChildrenClassName,
+  ButtonContentIconEndClassName,
+  ButtonContentIconStartClassName,
+  ButtonLoadingClassName,
+  ButtonLoadingIconClassName,
+} from "./Button.constants";
+import { Size, Kind } from "./Button.types";
+
+const Variables = css`
+  --button-color-bg: var(--ads-v2-color-bg);
+  --button-color-fg: var(--ads-v2-color-text);
+  --button-color-border: var(--ads-v2-color-border);
+  --button-font-weight: 600;
+  --button-font-size: 14px;
+  --button-icon-size: 16px;
+  --button-padding: 8px 12px;
+`;
+
+const Sizes = {
+  sm: css`
+    --button-font-weight: 500;
+    --button-font-size: 12px;
+    --button-padding: 4px 8px;
+    --button-icon-size: 12px;
+  `,
+  md: css`
+    --button-font-weight: 600;
+    --button-font-size: 14px;
+    --button-padding: 8px 12px;
+    --button-icon-size: 16px;
+  `,
+  lg: css`
+    --button-font-weight: 600;
+    --button-font-size: 16px;
+    --button-padding: 8px 16px;
+    --button-icon-size: 18px;
+  `,
+};
+
+const Kinds = {
```

Comment:
can't we call it `variant` instead of `kind`?

Reply:
You are doing this just to make old button API happy, you can use transformer. 
![image](https://user-images.githubusercontent.com/6636360/216753206-7d0cb02c-bd75-4a03-9fcc-286b9a836503.png)

Reply:
No kind is not to hold up with old API. Rather we took a decision on going with kind since it make more sense.